### PR TITLE
Airlocks don't feel clunky anymore

### PR DESCRIPTION
**Pull request by Hopekz**
This ancient airlock code hasn't been touched for 3 years.

Before:
------
9ptsUpqXBI

After:
------
EDIT:
-----
Fine-tuned the speed as it looked too early initially.

Here is how it looks now:
2Cgm6W0LwM

EDIT 2:
--------

Per Monster's suggestion did the same thing to closing doors.

**Before:**
gAS4sIeGQD

**After:**
a6F5eom0W0

#### Changelog

:cl:  Hopek
tweak: Makes airlocks feel less clunky by allowing you to walk through them when they're open on-screen vs when the full animation is done.
tweak: Makes airlocks feel even less clunky by allowing you to walk through them while closing when the on-screen animation has a gap VS you crashing into an invisible wall as soon as the animation starts.
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
hmm

**Comment by Hopekz** [comment]
EDIT:
-----
Fine-tuned the speed as it looked too early initially.

Here is how it looks now:
2Cgm6W0LwM

**Comment by johnnyboyz25** [comment]
Make fat people wait for the animation to finish

**Comment by monster860** [comment]
can you do the same with doors closing

**Comment by Hopekz** [comment]
> Make fat people wait for the animation to finish

Out of scope for this PR.
I'll PR it after this one is merged because it might be controversial.



> can you do the same with doors closing

Alright. Done.

**Before:**
gAS4sIeGQD

**After:**
a6F5eom0W0

**Comment by JamieD1** [comment]
Have you checked if its easier or harder to crush people as ai now?..

**Comment by Hopekz** [comment]
> Have you checked if its easier or harder to crush people as ai now?..

I'd imagine it's easier because now people have more opportunities to be under a door instead of being blocked by an invisible wall.

**Comment by jcatjmeow** [comment]
Personally I'd make it so you have to wait just a moment on the *closing*, cause right now the sprite player clips through the door for a bit. But I like this as is better than the default so idk

File: code/game/machinery/doors/airlock.dm
Comment (JamieD1) [inline code/game/machinery/doors/airlock.dm]:
```suggestion
	sleep(1)
```
Changing to test balance and ai closing

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**
